Build/Scripts/runTests.sh

```diff
@@ -649,6 +649,7 @@ case ${TEST_SUITE} in
         echo "Invalid -s option argument ${TEST_SUITE}" >&2
         echo >&2
         echo "${HELP}" >&2
+        ${CONTAINER_BIN} network remove ${NETWORK} >/dev/null
```

Comment:
I think we need to have the same Docker/Podman switch here that the Testing framework has:

https://github.com/TYPO3/testing-framework/pull/584/files